app/src/main/res/layout/fragment_detail.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        xmlns:tools="http://schemas.android.com/tools"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:id="@+id/detailLayout">
+
+    <TextView
+            android:text="@string/create_book_item"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:id="@+id/detail_frag_header"
+            android:textSize="30sp"
+            tools:layout_conversion_absoluteHeight="0dp"
+            tools:layout_conversion_absoluteWidth="0dp"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            android:layout_marginEnd="8dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            android:layout_marginTop="24dp"/>
+
+    <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="50dp"
+            android:text="@string/book_name"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            android:layout_marginEnd="8dp"
+            app:layout_constraintHorizontal_bias="0.51"
+            android:id="@+id/create_name"
+            android:layout_marginTop="8dp"
+            app:layout_constraintTop_toBottomOf="@+id/detail_frag_header"/>
+
+    <EditText
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:id="@+id/edit_name"
+            android:inputType="text"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            android:layout_marginEnd="8dp"
+            android:layout_marginStart="8dp"
+            android:hint="@string/name_book_hint"
+            app:layout_constraintTop_toBottomOf="@+id/create_name"
+            android:layout_marginTop="32dp"
+            app:layout_constraintHorizontal_bias="0.472"
+            android:autofillHints="" tools:targetApi="o"/>
```

Comment:
proc tu je android:autofillHints="" tools:targetApi="o"?

Reply:
to je to co mi zpusobilo zdvojeny layout, jdu opravit :)